Wikipedia/Code/Advanced Settings/InsertMediaAdvancedSettingsViewController.swift

```diff
@@ -142,6 +141,10 @@ extension InsertMediaAdvancedSettingsViewController: UITableViewDataSource {
         cell.textLabel?.textColor = cell.isUserInteractionEnabled ? theme.colors.primaryText : theme.colors.secondaryText
         cell.detailTextLabel?.textColor = theme.colors.secondaryText
     }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 50
```

Comment:
![Simulator Screen Shot - iPhone X - 2019-05-07 at 13 47 26](https://user-images.githubusercontent.com/741327/57321471-65cd3800-70cf-11e9-866d-59dd38228229.png)
^ breaks for larger dynamic type font sizes

Could try adjusting the cell margins? Otherwise I'd push back as I'm not sure it's worth a custom cell for the extra height.

Reply:
thanks for catching, I tried margins but they seem to have no effect

Reply:
not sure why but when setting `layoutMargins` or `directionalLayoutMargins` on `cell` in `cellForRowAt`, only `right` or `trailing` is applied, `top`, `left`/`leading`, `bottom` are ignored.

@carolynlimadeo would you be okay keeping the cell height as is? Right now, we're using a built-in cell, to increase the height we'd have to create a custom component